podaac/subsetter/subset.py

```diff
@@ -737,11 +737,10 @@ def subset_with_bbox(dataset, lat_var_names, lon_var_names, time_var_names, vari
                 var for var in dataset.data_vars.keys()
                 if var.startswith(f'{GROUP_DELIM}{var_prefix}')
             ]
-            if variables:
-                group_vars.extend([
-                    var for var in dataset.data_vars.keys()
-                    if var in variables and var not in group_vars and not var.startswith(tuple(lat_var_prefix))
-                ])
+            group_vars.extend([
```

Comment:
Why did you pull this out of the `if variables` block? Wouldn't this do nothing because the `if var in variables` would always be False?

Reply:
Re-added the if variables statement